packages/backend/src/index.ts

```diff
@@ -371,7 +364,8 @@ export function initIocContainer(
       peerService: await deps.use('peerService'),
       ratesService: await deps.use('ratesService'),
       streamServer: await deps.use('streamServer'),
-      ilpAddress: config.ilpAddress
+      ilpAddress: config.ilpAddress,
+      telemetry: await deps.use('telemetry')
```

Comment:
we should check if telemetry is defined here, otherwise `await deps.use('telemetry')` will throw

Reply:
![image](https://github.com/interledger/rafiki/assets/111863110/41a0bf66-f0f9-4bf2-b04f-36dc9e73a1a5)
It returns Promise<TelemetryService> | undefined, both of which are handled by the optionality of the telemetry dep in createConnectorService.
Or am i getting this wrong, deps.use('someThingThatIsNotRegistered') would throw indefeneatly?

Reply:
actually you were right...noticed it today. Resolved